esx_service/cli/vmdkops_admin.xml

```diff
@@ -0,0 +1,571 @@
+<?xml version="1.0"?>
+<plugin
+    xmlns="http://www.vmware.com/Products/ESX/6.0/esxcli/">
+    <!-- esxcli extension syntax version -->
+    <version>1.0.0</version>
+    <namespaces>
+        <namespace path="vdvs">
+            <description>vSphere Docker Volume Service admin CLI</description>
+        </namespace>
+        <namespace path="vdvs.volume">
+            <description>Manage vDVS volumes</description>
+        </namespace>
+        <namespace path="vdvs.policy">
+            <description>Configure and display storage policy information</description>
+        </namespace>
+        <namespace path="vdvs.vmgroup">
+            <description>Administer and monitor volume access control</description>
+        </namespace>
+        <namespace path="vdvs.vmgroup.vm">
+            <description>Add, removes, replaces and lists VMs in a vmgroup</description>
+        </namespace>
+        <namespace path="vdvs.vmgroup.access">
+            <description>Add or remove Datastore access and quotas for a vmgroup</description>
+        </namespace>
+        <namespace path="vdvs.config">
+            <description>Init and manage Config DB to enable quotas and access control [EXPERIMENTAL]</description>
+        </namespace>
+    </namespaces>
+    <commands>
+        <!-- Volume commands -->
+        <command path="vdvs.volume.ls">
+            <description>'List volumes'</description>
+            <input-spec>
+                <parameter name="vmgroup" type="string" required="false">
```

Comment:
Can vdvs.volume.ls specify --vmgroup option to only show volumes belong to a specific vmgroup? If yes, please add test for that.